docusaurus/extensions/catalog/fleet-workspace-ui.yaml

```diff
@@ -0,0 +1,6 @@
+name: Fleet workspace members UI
+author: Negashev
+authorUrl: https://negash.ru
+description: Management users for fleet workspace
+url: https://github.com/gorizond/fleet-workspace-ui
+icon: https://fleet.rancher.io/img/logo.svg
```

Comment:
Would you be able to use a gorizond icon?

Reply:
Yes